README.md

```diff
@@ -106,79 +102,71 @@ process.
 > It is designed to make programming web applications easier by making
 assumptions about what every developer needs to get started. It makes the
 assumption that there is a "best" way to do things, and it's designed to
-encourage that way - and in some cases to discourage alternatives. - Ruby on
+encourage that way - and in some cases to discourage alternatives. -- Ruby on
 Rails guide
 
-> Rails is a framework with lots of rules/conventions. Pay attention to the
+Rails is a framework with lots of rules/conventions. Pay attention to the
```

Comment:
And can ask them "what conventions do you know about already?"

Then link to cheat sheet